qiskit_ibm_runtime/qiskit_runtime_service.py

```diff
@@ -700,6 +702,7 @@ def save_account(
         verify: Optional[bool] = None,
         overwrite: Optional[bool] = False,
         channel_strategy: Optional[str] = None,
+        set_default_channel: Optional[bool] = True,
```

Comment:
Regarding the comment [here](https://github.com/Qiskit/qiskit-ibm-runtime/issues/954#issuecomment-1667794814) I think @jyu00 meant that there should be a way to set an entire account as the default account, not just a default channel. Then this default account would be used if no parameters were passed in to `QiskitRuntimeService()`. Users can still use the env var to set a default channel(original issue), but a default account would take precedence. 

Because it would be a little strange to have a parameter like `set_default_channel` passed in to `save_account` when the value isn't actually saved with the account, it's being saved as an extra value inside `qiskit-ibm.json`. There also wouldn't be a way to remove the `default_channel` value from the config json.

Reply:
You are probably right and my misunderstanding. I will wait for input from @jyu00  to make sure, before I make modifications.

Reply:
> Users can still use the env var to set a default channel(original issue), but a default account would take precedence.

@kt474 - you wrote here that the default account should take precedence over the the env var account. It seemed in the meeting today, you both thought the env var should take precedence. Any final opinion?

Reply:
I am copying the existing documentation here:

     An account is selected in the following order:

            - Account with the input `name`, if specified.
            - Default account for the `channel` type, if `channel` is specified but `token` is not.
            - Account defined by the input `channel` and `token`, if specified.
            - Account defined by the environment variables, if defined.
            - Default account for the ``ibm_cloud`` account, if one is available.
            - Default account for the ``ibm_quantum`` account, if one is available.
So I guess this is consistent with what @kt474 wrote above and with the order I specified in my PR.

Reply:
But now I am even more confused: there is a default account defined for `ibm_cloud` and a default for `ibm_quantum`. So I think it would make sense to simply define the `default_channel`, and then we would select the account according to the `default_channel`. What do you think?
The other option would be to have an additional account, name "default" which would probably be a copy of one of the accounts above. Or else to add a field to the account, something like "is_default_account", but I think this is more confusing.

Reply:
Yes, I think that's a good solution - we can have a `default_channel` and then it would just select the appropriate default channel account. So setting the env variable `QISKIT_IBM_CHANNEL` will allow users to to set their default channel to the quantum channel. 

If we really wanted to, we could add another "default" flag but that would overcomplicate the already very complicated account management. This wasn't part of the original issue so I'm ok with not having this

Reply:
So I think I will go with adding a "is_default_account" field to the account that is default. It will require going over all the accounts to check which one is default, but that should be so bad, because normally people will only keep at most a handful of accounts.